src/native/libs/configure.cmake

```diff
@@ -581,23 +581,22 @@ if(CLR_CMAKE_TARGET_IOS)
     # Manually set results from check_c_source_runs() since it's not possible to actually run it during CMake configure checking
     unset(HAVE_SHM_OPEN_THAT_WORKS_WELL_ENOUGH_WITH_MMAP)
     unset(HAVE_ALIGNED_ALLOC)   # only exists on iOS 13+
```

Comment:
@kotlarmilos What is our current minimum support iOS version? Can this block be deleted?

Reply:
Do we know why we support such an old version of iOS? Apple starts considering them vintage after they've no longer been on sale for 5 years and obsolete after 7 years: https://support.apple.com/en-us/HT201624

The iPhone 6 is the newest vintage phone supported and it supports iOS 12.5.7 as its latest.
While the iPhone 7 is the oldest non-vintage phone, and that shipped with iOS 10 but currently runs iOS 15.8

For macOS, 10.15 hasn't had a security update in 15 months (https://support.apple.com/en-us/102769). While 11 still got one less than 2 months ago (https://support.apple.com/en-us/HT211896#macos11710)

Reply:
I believe it depends on Xamarin/Maui versioning. @rolfbjarne What is the versioning policy for Xamarin?

Reply:
We had discussed bumping the min version to 12 in .net 9. Unless @akoeplinger found some kind of holdup?

Reply:
I have opened https://github.com/dotnet/runtime/issues/94345 on this.